docs/contracts/contracts/LSP0ERC725Account/LSP0ERC725Account.md

```diff
@@ -0,0 +1,1485 @@
+#### Returns
+
+| Name      |   Type    | Description                                            |
+| --------- | :-------: | ------------------------------------------------------ |
+| `results` | `bytes[]` | An array of values returned by the executed functions. |
+
+### execute
+
+:::note Links
+
+- Specification details in [**LSP-0-ERC725Account**](https://github.com/lukso-network/lips/tree/main/LSPs/LSP-0-ERC725Account.md#execute)
+- Solidity implementation in [**LSP0ERC725Account**](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP0ERC725Account/LSP0ERC725Account.sol)
+- Function signature: `execute(uint256,address,uint256,bytes)`
+- Function selector: `0x44c028fe`
+
+:::
+
+```solidity
+function execute(
+  uint256 operationType,
+  address target,
+  uint256 value,
+  bytes data
+) external payable returns (bytes);
+```
+
+_Calling address `target` using `operationType`, transferring `value` wei and data: `data`. _
+
+Generic executor function to:
+
+- send native tokens to any address.
+
+- interact with any contract by passing an abi-encoded function call in the `data` parameter.
+
+- deploy a contract by providing its creation bytecode in the `data` parameter.
+
+<blockquote>
+
+**Requirements:**
+
+- If a `value` is provided, the contract MUST have at least this amount in its balance to execute successfully.
+- If the operation type is `STATICCALL` or `DELEGATECALL`, `value` SHOULD be 0.
+- `target` SHOULD be `address(0)` when deploying a contract.
+- MUST pass when called by the owner or by an authorised address that pass the verification check performed on the owner accordinng to [LSP-20-CallVerification] specification.
+
+</blockquote>
+
+<blockquote>
+
+**Emitted events:**
+
+- [`Executed`](#executed) event, when a call is executed under `operationType` 0, 3 and 4
+- [`ContractCreated`](#contractcreated) event, when a contract is created under `operationType` 1 and 2
+- [`ValueReceived`](#valuereceived) event when receiving native tokens.
+
+</blockquote>
+
+#### Parameters
+
+| Name            |   Type    | Description                                                                                           |
+| --------------- | :-------: | ----------------------------------------------------------------------------------------------------- |
+| `operationType` | `uint256` | The operation type used: CALL = 0; CREATE = 1; CREATE2 = 2; STATICCALL = 3; DELEGATECALL = 4          |
+| `target`        | `address` | The address of the EOA or smart contract. (unused if a contract is created via operation type 1 or 2) |
+| `value`         | `uint256` | The amount of native tokens to transfer (in Wei)                                                      |
+| `data`          |  `bytes`  | The call data, or the creation bytecode of the contract to deploy if `operationType` is `1` or `2`.   |
+
+#### Returns
+
+| Name |  Type   | Description |
+| ---- | :-----: | ----------- |
+| `0`  | `bytes` | -           |
```

Comment:
Please describe the return parameter here.

Reply:
This should be described on the `IERC725X` interface as we  inherit it here.